CHANGELOG.md

```diff
@@ -7,6 +7,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [2.0.4] - 2022-04-22
```

Comment:
@000-ooo-000 

You can ommit the this line. Just add your changes under the unreleased sections. Will automatically be copied over by the release pipeline :-)

Reply:
Ommitted line